packages/server/package.json

```diff
@@ -143,7 +143,7 @@
     "jest": "29.7.0",
     "jest-openapi": "0.14.2",
     "jest-runner": "29.7.0",
-    "jest-serial-runner": "1.2.1",
+    "nock": "^13.5.4",
```

Comment:
Generally we have always pinned package versions - the `yarn.lock` does sort this for us anyway, but for consistency probably worth removing the `^`.

Reply:
Ahh I ran `yarn add --dev nock` and this was what it put in by default. Will change.

Reply:
> Ahh I ran `yarn add --dev nock` and this was what it put in by default. Will change.

Yeap by default this is what happens, sadly theres no real way to change the default `add` functionality, we just tend to pin!